platform/commonUI/general/res/sass/_main.scss

```diff
@@ -76,9 +76,11 @@
 @import "items/item";
 @import "mobile/item";
 
-
 /********************************* TO BE MOVED */
 @import "autoflow";
 @import "features/imagery";
 @import "features/time-display";
 @import "widgets";
+
+/********************************* APP STARTUP */
+@import "app-start";
```

Comment:
@charlesh88 - Add new line at end of file

Reply:
@dtailor90 Pushed up requested change.